test/unit/ProductDataTest.php

```diff
@@ -51,10 +52,10 @@ public function testDefaultValuesAreCorrect()
         self::assertEquals(ProductVisibility::CATALOG_SEARCH, $product->getVisibility());
         self::assertEquals(null, $product->getPrice());
         self::assertEquals(null, $product->getWeight());
+        self::assertEquals(null, $product->getMediaGalleryEntries());
         self::assertEquals(ProductTypeId::SIMPLE, $product->getTypeId());
         self::assertEquals(ProductData::DEFAULT_ATTRIBUTE_SET_CODE, $product->getAttributeSetCode());
         self::assertTrue(($product->getCustomAttributes())->isEmpty());
-        self::assertTrue(($product->getMediaGalleryEntries())->isEmpty());
```

Comment:
Instead of removing this assertion I think it is better to just assert null just to keep it explicit since it is not being tested anywhere else.

Reply:
ignore this one.,  it saw this as removed but this is exacly what I wanted.